slurmci-test.toml

```diff
@@ -38,7 +38,7 @@ cpu_gpu = [
   { file = "test/Numerics/LinearSolvers/bandedsystem.jl", slurmargs = ["--ntasks=3", "--time=02:00:00"], args = [] },
   { file = "test/Numerics/Mesh/interpolation.jl", slurmargs = ["--ntasks=3", "--time=02:00:00"], args = [] },
   { file = "test/Ocean/ShallowWater/GyreDriver.jl", slurmargs = ["--ntasks=1"], args = [] },
-  { file = "tutorials/Numerics/DGmethods/nonnegative.jl", slurmargs = ["--ntasks=3"], args = [] },
+  { file = "tutorials/Numerics/DGmethods/nonnegative.jl", slurmargs = ["--ntasks=3"], args = ["--output-dir=output/dgmethods_nonnegative"] },
```

Comment:
This might have to be moved to have separate ones on gpu and cpu, since otherwise both jobs will use the same directory.